app/components/ChannelForm/ChannelForm.js

```diff
@@ -51,7 +51,7 @@ const ChannelForm = ({
       className={styles.modal}
     >
       <div onClick={closeChannelForm} className={styles.modalClose}>
-        <FaClose className={styles.close} />
```

Comment:
Looks like styles.close is unused after this removal. You can drop it from the scss as well.

Reply:
Right! Thanks for the tip, Webstorm didn't show me that.

Reply:
It's not in the SCSS at all, that's why I removed it https://github.com/pajasevi/zap-desktop/blob/95872d8ea3ed76f50ece53173e0871f2f83269b6/app/components/ChannelForm/ChannelForm.scss